server/public/model/job.go

```diff
@@ -39,7 +39,7 @@ const (
 	JobTypeS3PathMigration               = "s3_path_migration"
 	JobTypeCleanupDesktopTokens          = "cleanup_desktop_tokens"
 	JobTypeDeleteEmptyDraftsMigration    = "delete_empty_drafts_migration"
-	JobTypeRefreshPostStats              = "refresh_post_stats"
+	JobTypeRefreshMaterializedViews      = "refresh_materialized_views"
```

Comment:
Just checking that renaming a job type isn't going to cause compatibility issues given it's technically a breaking change.

Reply:
Well, the model package is now under 0.x versioning. So this is no longer a breaking change.